### PR TITLE
Add lock and unlock record to record access dialog box

### PR DESCRIPTION
**Pull request by elsiosanchez**
## Feature
Add lock and unlock record to record access dialog box
#### Screenshot or Gif

cal

#### Other relevant information
- Your OS: Debian 9
- Web Browser: Chrome
- Node.js version: 10.0.1

### CODE REVIEW

**Comment by oscarinak** [comment]
The pull verification was carried out in the following service "https://deploy-preview-805--stoic-lamport-68decf.netlify.app/" where it does not comply with what is established in pull since at the interface level it is disordered

Verificación_Pull_805

**Comment by oscarinak** [comment]
Fails

File: src/lang/ADempiere/es.js [hunk 1]
```diff
@@ -316,10 +316,12 @@ export default {
     noDescription: 'Sin Descripción',
     recordAccess: {
       actions: 'Acceso a Registros',
-      hideRecord: 'Ocultar Registro',
-      recordDisplay: 'Permitir Visualización del Registro',
```
Comment (yamelsenih) [inline src/lang/ADempiere/es.js:320]: Change this by **displayRecord**

File: src/lang/ADempiere/en.js [hunk 2]
```diff
@@ -341,10 +341,12 @@ export default {
     noDescription: 'No Description',
     recordAccess: {
       actions: 'Record Access',
-      hideRecord: 'Hide Record',
-      recordDisplay: 'Allow Record Display',
```
Comment (yamelsenih) [inline src/lang/ADempiere/en.js:345]: Change this by displayRecord

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/RecordAccess/recordAccess.js [hunk 3]
```diff
@@ -79,7 +79,7 @@ export default {
     includedList: {
       get() {
         if (this.recordAccess.roles) {
-          return this.recordAccess.roles.filter(role => role.isLocked)
+          return this.recordAccess.roles.filter(role => role.isRoleConfi)
```
Comment (yamelsenih) [inline src/components/ADempiere/RecordAccess/recordAccess.js]: Change this by **role.isRoleConfig**

File: src/components/ADempiere/RecordAccess/recordAccess.js [hunk 4]
```diff
@@ -105,13 +105,16 @@ export default {
         access.availableRoles.forEach(role => {
           this.recordAccess.roles.push({
             ...role,
-            isLocked: false
+            isRoleConfi: false,
```
Comment (yamelsenih) [inline src/components/ADempiere/RecordAccess/recordAccess.js]: Change this by **role.isRoleConfig**

File: src/components/ADempiere/RecordAccess/recordAccess.js [hunk 5]
```diff
@@ -105,13 +105,16 @@ export default {
...
+            isLocked: role.isExclude
           })
         })
         access.currentRoles.forEach(role => {
-          this.recordAccess.roles.find(availableRole => availableRole.roleId === role.roleId).isLocked = true
+          this.recordAccess.roles.find(availableRole => availableRole.roleId === role.roleId).isLocked = role.isExclude
+          this.recordAccess.roles.find(availableRole => availableRole.roleId === role.roleId).isRoleConfi = true
```
Comment (yamelsenih) [inline src/components/ADempiere/RecordAccess/recordAccess.js]: Change this by **role.isRoleConfig**

File: src/components/ADempiere/RecordAccess/recordAccess.js [hunk 6]
```diff
@@ -153,7 +156,7 @@ export default {
       index,
       element
     }) {
-      this.recordAccess.roles[index].isLocked = false
+      this.recordAccess.roles[index].isRoleConfi = false
```
Comment (yamelsenih) [inline src/components/ADempiere/RecordAccess/recordAccess.js]: Change this by **role.isRoleConfig**

File: src/components/ADempiere/RecordAccess/recordAccess.js [hunk 7]
```diff
@@ -143,7 +146,7 @@ export default {
       index,
       element
     }) {
-      this.recordAccess.roles[index].isLocked = true
+      this.recordAccess.roles[index].isRoleConfi = true
```
Comment (yamelsenih) [inline src/components/ADempiere/RecordAccess/recordAccess.js]: Change this by **role.isRoleConfig**

**Review by yamelsenih — CHANGES_REQUESTED** [review]

**Pull request merged**